src/extractTime.ts

```diff
@@ -153,6 +153,9 @@ export function parseTimeFromLog(log: string, logType: CloudWatchLogTypes): numb
   switch (logType) {
     case CloudWatchLogTypes.app:
       return extractAppLogTime(log)
+    case CloudWatchLogTypes.adot:
+      // Adot has many different time formats, so we'll rely on the cloudwatch timestamp instead
+      return undefined
```

Comment:
We might want to remove `console.log('Using cloudwatch event time')` on line 129, as it will be logged for every adot log line.